.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
In general, I don't think we should commit editor specific files to the repo. Is using the local TS not the default behavior?

Reply:
I agree, and sadly it is *not*. (Which boggles my mind.)

Reply:
Uh, wut. How does this make sense? Have we searched for a bug report? This seems fundamentally invalid to me... 🤔

Reply:
You and me and everyone else. It's a standing WHUT among VS Code TS users.

Reply:
I still don't like this 😡, but I guess it isn't worth pwning everyone that works here based on my "editor puristism".

Reply:
FWIW, I'm entirely with you. It ticks me off every repo I have to add this to.